cli/crates/server/src/udf_builder.rs

```diff
@@ -18,20 +18,29 @@ async fn run_command<P: AsRef<Path>>(
     environment: &[(&'static str, &str)],
 ) -> Result<(), JavascriptPackageManagerComamndError> {
     let command_string = format!("{command_type} {}", arguments.iter().format(" "));
-
-    trace!("running '{command_string}'");
+    let current_directory = current_directory.as_ref();
+    if !current_directory.exists() {
+        return Err(JavascriptPackageManagerComamndError::WorkingDirectoryNotFound(
+            current_directory.to_owned(),
+        ));
+    }
+    debug!("running '{command_string}'");
```

Comment:
We don't really use anything other than `trace!` at the moment, I'd keep it consistent and then possibly change some of the invocations if we want to add levels

Reply:
sure, changing it.